app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,15 @@
+class AdminController < ApplicationController
+
+  def index
+    unless current_user_is_admin?
+      flash[:error] = "You do not have sufficient permissions to view this page"
```

Comment:
I think there is, but I think it only works if you mix in HydraController. I'm not sure what all that brings in.